src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -211,7 +248,18 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
             ->setDefault(self::ALIGN_VERTICAL)
         ;
 
-        return new FixerConfigurationResolver([$tags->getOption(), $align->getOption()]);
+        $spacing = new FixerOptionBuilder(
+            'spacing',
+            'Spacing between tag, hint, comment, signature, etc. You can set same spacing for all tags using a positive integer '.
+            'or different spacings for different tags using an associative array of positive integers '.
+            '`[\'tagA\' => spacingForA, \'tagB\' => spacingForB]`.'
```

Comment:
Minor: can we use heredoc instead of concatenated string? Or maybe single string like for `tags` (even if it's longer than suggested line width)?

Reply:
Single string applied.